### PR TITLE
Added tests for Event.java. Fixed a typo in CalendarEvent.java.

### PR DESCRIPTION
**Pull request by Fourstarfish**
Tests passed successfully.

### CODE REVIEW

File: tests/entityTests/EventTest.java [hunk 1]
```diff
@@ -1,4 +1,103 @@
...
+        assertTrue("Staff should be added successfully", event.addStaff(employee));
+        assertFalse("Same staff should not be added again", event.addStaff(employee));
+    }
+
+    @Test
+    public void testRemoveStaff() {
+        event.addStaff(employee);
+        assertTrue("Staff should be removed successfully", event.removeStaff(employee));
+        assertFalse("Non-existent staff should not be removed", event.removeStaff(employee));
+    }
```
Comment (xubosen) [inline tests/entityTests/EventTest.java:45]: We might want to check that the staff no longer contains any of the sessions and all the sessions no longer contain the staff.

File: tests/entityTests/EventTest.java [hunk 2]
```diff
@@ -1,4 +1,103 @@
...
+        assertTrue("Session should be added successfully", event.addSession(session1));
+        assertFalse("Same session should not be added again", event.addSession(session1));
+    }
+
+    @Test
+    public void testRemoveSession() {
+        event.addSession(session2);
+        assertTrue("Session should be removed successfully", event.removeSession(session2));
+        assertFalse("Non-existent session should not be removed", event.removeSession(session2));
+    }
```
Comment (xubosen) [inline tests/entityTests/EventTest.java:58]: We might want to check that the removed session now has null for its event attribute and also that none of the employees have this session in their map of sessions.

**Review by xubosen — COMMENTED** [review]
See more details in in-line comments. Mostly good, with just a few details missing.

**Review by alexgphan — APPROVED** [review]
Current tests work but we still need to add the tests relating to the above possible errors.

**Pull request merged**